doc/source/jupyterhub/customizing/user-management.md

```diff
@@ -49,7 +49,7 @@ are being used as expected.
 ```
 
 ```{note}
-There is a separate culling service for Jupyter Notebook that provides more fine-grained control over culling behavior for individual notebooks. That culling behavior has not yet been extended to Jupyter Lab [see here](https://github.com/jupyterlab/jupyterlab/issues/6893). However, when running processes in Classic Notebook, these settings, if passed to user pods, can control culling on active processes regardless of web socket connection. See the [Jupyter Notebook Docs](https://jupyter-notebook.readthedocs.io/en/stable/config.html#options) for more info, specifically options: `MappingKernelManager.cull_idle_timeout`, `MappingKernelManager.cull_interval`, `MappingKernelManager.cull_connected`, and `NotebookApp.shutdown_no_activity_timeout`.
+There is a separate culling service for Jupyter Notebook that provides more fine-grained control over culling behavior for individual kernels (running notebooks or terminals). That culling behavior has not yet been extended to Jupyter Lab [see here](https://github.com/jupyterlab/jupyterlab/issues/6893). However, when running processes in Classic Notebook, these settings, if passed to user pods, can control culling on active processes regardless of web socket connection. See the [Jupyter Notebook Docs](https://jupyter-notebook.readthedocs.io/en/stable/config.html#options) for more info, specifically options: `MappingKernelManager.cull_idle_timeout`, `MappingKernelManager.cull_interval`, `MappingKernelManager.cull_connected`, and `NotebookApp.shutdown_no_activity_timeout`.
```

Comment:
`for individual kernels (running notebooks or terminals)` I know that you can control Terminal session culling as of Jupyter Notebook v6.1. So I believe this is accurate but I am not certain it is exhaustive.